Adding Contributors to README.md (Front-end)

hariom1625 opened the pull request:
fixes #88 
<Do not delete this template>
Description: <provide a description> Added Contributors list in the README.md .
In the picture attached below, there are two types of layout present, one with an emoticon below the name and one without, I need your views which one looks better. The emoticons can be varied.

#### Testimonial
![contributors](https://user-images.githubusercontent.com/66957239/94315713-ac5f5300-ffa0-11ea-9806-6fb1eb2f8895.png)

hariom1625 committed:
update readme.md contributors

ankiiitraj commented:
One last this is you can squash these commit and then push again! 
And you're good to go 🎉

tusharjain0022 commented:
I think without the emoticons it would look nice, and we don't have to bother much about which emoticons to be used with whom otherwise both looks cool to me! 😄.

hariom1625 committed:
parent 4534cbeaec23aaa07c385a51690b290459c0b529
author hariom1625 <hariomyadav.btech.cs19@iiitranchi.ac.in> 1601070760 +0530
committer hariom1625 <hariomyadav.btech.cs19@iiitranchi.ac.in> 1601070760 +0530

update readme.md

anktrj commented on .all-contributorsrc:
```diff
@@ -0,0 +1,105 @@
+      "contributions": [
+        "code"
+      ]
+<<<<<<< HEAD
```
whoops . . . forgot these?

hariom1625 replied:
Don't know-how, it isn't visible in my code editor, showing only on GitHub.😕
Will fix !

hariom1625 committed:
Update .all-contributorsrc

hariom1625 commented:
> One last this is you can squash these commit and then push again!
> And you're good to go 🎉

Done and Thank you for suggesting this :-)

hariom1625 commented:
> I think without the emoticons it would look nice, and we don't have to bother much about which emoticons to be used with whom otherwise both looks cool to me! 😄.

Right 💯
